event-log/src/main/scala/io/renku/eventlog/Microservice.scala

```diff
@@ -57,19 +58,19 @@ object Microservice extends IOMicroservice {
   protected implicit override def timer: Timer[IO] = IO.timer(executionContext)
 
   override def run(args: List[String]): IO[ExitCode] = for {
-    transactorResource <- new EventLogDbConfigProvider[IO](args) map DbTransactorResource[IO, EventLogDB]
+    transactorResource <- new EventLogDbConfigProvider[IO]() map SessionPoolResource[IO, EventLogDB]
```

Comment:
Maybe `transactorResource` should be renamed to `sessionPoolResource`? Probably the same could be done on token-repository's Microservice